<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeamCarveWndClass</class>
 <widget class="QMainWindow" name="SeamCarveWndClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>553</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Seam Carving - Yan Jun</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GLImageViewer" name="glviewer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuEnergy_Measure">
     <property name="title">
      <string>Energy Measure</string>
     </property>
     <addaction name="actionEnergyGradient"/>
     <addaction name="actionEnergySaliency"/>
     <addaction name="actionEnergyGradSaliency"/>
    </widget>
    <widget class="QMenu" name="menuCost_Matrix_Computation">
     <property name="title">
      <string>Cost Matrix Computation</string>
     </property>
     <addaction name="actionMatrixDP"/>
     <addaction name="actionMatrixParallel"/>
    </widget>
    <addaction name="menuEnergy_Measure"/>
    <addaction name="menuCost_Matrix_Computation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDrawProtect"/>
   <addaction name="actionDrawRemove"/>
   <addaction name="separator"/>
   <addaction name="actionResetSize"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save...</string>
   </property>
  </action>
  <action name="actionDrawProtect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Protect</string>
   </property>
   <property name="toolTip">
    <string>Draw Protecting Region</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDrawRemove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Draw Removal Region</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionResetSize">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Image Size</string>
   </property>
  </action>
  <action name="actionEnergyGradient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient</string>
   </property>
  </action>
  <action name="actionEnergySaliency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Saliency Map</string>
   </property>
  </action>
  <action name="actionEnergyGradSaliency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient*Saliency</string>
   </property>
  </action>
  <action name="actionMatrixDP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dynamic Programming</string>
   </property>
  </action>
  <action name="actionMatrixParallel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Greedy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="seamcarvewnd.qrc"/>
 </resources>
 <connections/>
</ui>
